src/css/components/pagination.css

```diff
@@ -16,16 +16,39 @@
   line-height: calc(var(--base-spacing) * 6);
   min-width: calc(var(--base-spacing) * 6);
   min-height: calc(var(--base-spacing) * 6);
+  border-radius: calc(var(--base-spacing) * 3);
+  background: transparent;
+  transition: background 0.15s ease-in-out, color 0.15s ease-in-out;
```

Comment:
Histoire d'utiliser toujours la même façon d'écrire des chiffres entre 0 et 1, on peut :

```suggestion
  transition: background .15s ease-in-out, color .15s ease-in-out;
```

Reply:
Vu la tradition je vais laisser les `0`
<img width="941" alt="image" src="https://user-images.githubusercontent.com/1781070/96722214-6b930800-13ad-11eb-9bb7-7765ccb1f85a.png">

Reply:
Ah oui, effectivement, la tradition est de le mettre quand il s'agit de secondes mais pas de rem 🤷 Ça me va !